src/views/partials/settings.js

```diff
@@ -1,5 +1,183 @@
+    return 1
+  }
+
+  return a.email.localeCompare(b.email)
+})
+
+const createEmailList = (emails, breachCounts) => `
+  <ul class='settings-email-list'>
+    ${getSortedEmails(emails)
+      .map(email => createEmailItem(email, breachCounts))
+      .join('')}
+  </ul>
+`
+
+const optionInput = (csrfToken, { isChecked, option }) => `
+  <input
+    checked='${isChecked}'
+    class='js-settings-alert-options-input'
+    data-alert-option='${option}'
+    data-csrf-token='${csrfToken}'
+    name='settings-alert-options'
+    type='radio'
+  >
+`
+
+const alertOptions = csrfToken => `
+  <div class='settings-alert-options'>
+    <label class='settings-radio-input'>
+    ${optionInput(csrfToken, {
+      isChecked: true,
+      option: 0
+    })}
+    <span class='settings-radio-label'>
+      ${getMessage('settings-alert-preferences-option-one')}
+    </span>
+  </label>
+
+  <label class='settings-radio-input'>
+    ${optionInput(csrfToken, {
+      isChecked: false,
+      option: 1
+    })}
+    <span class='settings-radio-label'>
+      ${getMessage('settings-alert-preferences-option-two')}
+    </span>
+  </label>
+  </div>
+`
+
+const createAddEmailModal = limit => `
+  <dialog id='js-settings-modal' class='settings-email-modal'>
+    <button id='js-settings-close'>
+      <img src='/images/icon-close.svg'>
+    </button>
+    <img src='/images/settings-dialog-email.svg'>
+
+    <h3 class='settings-section-title'>
+      ${getMessage('settings-email-dialog-title')}
+    </h3>
+    <div id='js-settings-modal-content'>
+      ${getMessage('settings-email-limit-info', { limit })}
+      ${getMessage('settings-add-email-text')}
+    </div>
+
+    <div id='js-settings-modal-controls'>
+      <label>
+        ${getMessage('settings-email-input-label')}
+        <input id='js-settings-email-modal-input' type='text'>
+      </label>
+      <button id='js-settings-modal-send-verification' class='primary'>
+        ${getMessage('settings-send-email-verification-button')}
+      </button>
+    </div>
+  </dialog>
+`
+
+export const settings = data => {
+  const { breachCounts, csrfToken, emails, limit } = data
+
+  return `
+    <div id='js-settings' class='settings' data-csrf-token='${csrfToken}'>
+      <h2 class='settings-title'>${getMessage('settings-page-title')}</h2>
+
+      <div class='settings-content'>
+        <!-- Breach alert preferences -->
+        <section>
+          <h3 class='settings-section-title'>
+            ${getMessage('settings-alert-preferences-title')}
+          </h3>
+          ${alertOptions(csrfToken)}
+        </section>
+
+        <hr>
+
+        <!-- Monitored email addresses -->
+        <section>
+          <h3 class='settings-section-title'>
+            ${getMessage('settings-email-list-title')}
+          </h3>
+          <p>${getMessage('settings-email-limit-info', { limit })}</p>
+
+          ${createEmailList(emails, breachCounts)}
+          <button
+            id='js-settings-add-email-opener'
```

Comment:
See note about ID attribute global vars and if we should avoid for non-form elements.